omero/developers/Python.rst

```diff
@@ -310,6 +341,31 @@ Read data
                 well.getImage(index).getName(),\
                 well.getImage(index).getId()
 
+-  **List all annotations on an object. Get text from tags**
+
+::
+
+    for ann in project.listAnnotations():
+        print ann.getId(), ann.OMERO_TYPE,
+        print " added by ", ann.link.getDetails().getOwner().getOmeName()
+        if ann.OMERO_TYPE == omero.model.TagAnnotationI:
+            print "Tag value:", ann.getTextValue()
+
+-  **Get Links between Objects and Annotations**
+
+::
+
+    # Find Images linked to Annotation(s), move them to another Annotation
```

Comment:
"... move them (images) to another Annotation." -> not sure what this says: Maybe "find images by Annotations, unlink the present annotations from images thus found, then link them to other Annotations" ?